webui/react/src/stores/workspaces.tsx

```diff
@@ -129,6 +129,8 @@ class WorkspaceStore extends PollingStore {
   public readonly boundResourcePools = (workspaceId: number) =>
     this.#boundResourcePools.select((map) => map.get(workspaceId)?.toJS());
 
+  public readonly boundResourcePoolsMap = () => this.#boundResourcePools.readOnly();
```

Comment:
Does that need to be a function? 
why not like this?
```suggestion
  public readonly boundResourcePoolsMap = this.#boundResourcePools.readOnly();
```